hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineManagerImpl.java

```diff
@@ -535,9 +548,11 @@ public void closeStalePipelines(DatanodeDetails datanodeDetails) {
             pipelinesWithStaleIpOrHostname.size());
     pipelinesWithStaleIpOrHostname.forEach(p -> {
       try {
-        LOG.info("Closing the stale pipeline: {}", p.getId());
-        closePipeline(p, false);
-        LOG.info("Closed the stale pipeline: {}", p.getId());
+        final PipelineID id = p.getId();
+        LOG.info("Closing the stale pipeline: {}", id);
```

Comment:
Nit: I am not sure we need this log message. We have a message below for "closed" and then exception handler has a message for "failed to close".